source/wp-content/themes/wporg-developer-2023/patterns/article-sidebar.php

```diff
@@ -8,7 +8,13 @@
 ?>
 
 <!-- wp:wporg/sidebar-container -->
-	<!-- wp:search {"label":"<?php esc_attr_e( 'Search', 'wporg' ); ?>","showLabel":false,"width":100,"widthUnit":"%","buttonText":"<?php esc_attr_e( 'Search', 'wporg' ); ?>","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control","borderColor":"light-grey-1","placeholder":"<?php esc_attr_e( 'Search in the documentation', 'wporg' ); ?>"} /-->
 
-	<!-- wp:wporg/table-of-contents /-->
+	<!-- wp:group {"tagName":"article","style":{"spacing":{"padding":{"top":"var:preset|spacing|20"}}}} -->
+	<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--20)">
```

Comment:
This adjustment should be temporary until we can move the search field to the top of the Documentation pages too, then the margin-top of the sidebar-container should be changed instead.